### PR TITLE
Clean up UI

### PR DESCRIPTION
**Pull request by jauschalley**
*What does this PR do?*
This is just some text removal from the UI.

*Jira ticket number?*
No ticket

*Changes*

- Add generic name to header for demos
- Remove todo showing in the UI

*Testing*

- [ ] Step 1 : Login and see "User Name" in right side of header
- [ ] Step 2: Click on a project and see that the todo message is no longer present above "System Components"

### CODE REVIEW

**Review by amyd11 — APPROVED** [review]

File: src/stub.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const UserStub = {
   id: 1,
-  username: "Dick",
```
Comment (rmoore-theta) [inline src/stub.js:3]: How is this a cleanup? This should be replaced with real data as part of the authentication.

File: src/stub.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  username: "User Name",
   first_name: "",
   last_name: "",
   email: "r.moore@theta-llc.com",
```
Comment (jwhite3) [inline src/stub.js:6]: Probably should also give a fake email as well, but not necessarily a blocking change.

**Review by jwhite3 — APPROVED** [review]

File: src/stub.js (hunk 1 above)
Comment (jwhite3) [reply src/stub.js:3]: I think that this is being handled in @amyd11 's PR, https://github.com/CMSgov/blueprint_ui/pull/62

Comment (rmoore-theta) [reply src/stub.js:3]: Exactly @jwhite3 this entire file will be replaced with real data at that time.

**Pull request closed without merging**